pythonforandroid/recipes/numpy/__init__.py

```diff
@@ -1,17 +1,17 @@
 from pythonforandroid.recipe import Recipe, MesonRecipe
-from pythonforandroid.logger import error
 from os.path import join
 import shutil
 
 NUMPY_NDK_MESSAGE = "In order to build numpy, you must set minimum ndk api (minapi) to `24`.\n"
 
 
 class NumpyRecipe(MesonRecipe):
-    version = 'v1.26.5'
+    version = 'v2.3.0'
     url = 'git+https://github.com/numpy/numpy'
-    hostpython_prerequisites = ["Cython>=3.0.6"]  # meson does not detects venv's cython
+    hostpython_prerequisites = ["Cython==3.0.6"]  # meson does not detects venv's cython
```

Comment:
Do we have to be that restrictive? Maybe it doesn't work with too recents Cython versions, but in this case maybe we should use a range at least